src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,29 @@
+import "./Home.css";
+import {Navbar,Product,Footer} from "../../components/index";
+import {Hero} from "./HeroSection/Hero";
+import {CategoryList} from "./CategoryList/CategoryList";
+import { useStateContext } from "../../context/stateContext";
+
+const Home = () => {
+    const {state}=useStateContext();
+    const {products}=state;
+    console.log(products);
+    return (
+        <div className="Home">
+            <Navbar />
```

Comment:
For Navbar, you can import this Navbar to App.js so you don't need to import this to every page.